x-pack/plugins/cross_cluster_replication/public/app/components/auto_follow_pattern_form.js

```diff
@@ -301,13 +308,13 @@ export class AutoFollowPatternFormUI extends PureComponent {
         />),
         remoteClusterNotConnectedNotEditable: () => (<FormattedMessage
           id="xpack.crossClusterReplication.autoFollowPatternForm.currentRemoteClusterNotConnectedCallOutDescription"
-          defaultMessage="You need to connect it before editing this auto-follow pattern. Edit the remote cluster to
-            fix the problem."
+          defaultMessage="The remote cluster must be connected to edit this auto-follow pattern."
         />),
```

Comment:
Can't edit auto-follow pattern because remote cluster 'test' is not connected.  

You can address this by editing the remote cluster.

Edit remote cluster